CHANGELOG.md

```diff
@@ -90,6 +90,10 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 * Fixed bash and zsh shell completion when completing aliases of multiple arguments.
   [#5377](https://github.com/jj-vcs/jj/issues/5377)
 
+* Work around a git issue on macOS that could cause `fetch` operations to hang if the
+  macOS `fsmonitor` feature is enabled.
+  [#6440](https://github.com/jj-vcs/jj/issues/6440)
+
```

Comment:
nit: you'll need to update the changelog after the release today.

Reply:
No problem, I'll do so once the release happens!